devicetypes/smartthings/aeon-multisensor-6.src/aeon-multisensor-6.groovy

```diff
@@ -309,6 +313,8 @@ def zwaveEvent(physicalgraph.zwave.commands.notificationv3.NotificationReport cm
 				break
 			case 3:
 				result << createEvent(name: "tamper", value: "detected", descriptionText: "$device.displayName was tampered")
+				// Clear the tamper alert after 10s. This is a temporary fix for the tamper attribute until local execution handles it
+				runIn(10, clearTamper)
```

Comment:
Should we also remove the clear in `case 0`?

Reply:
We should `unschedule(clearTamper)` before this, in case there are multiple alerts in the time frame.

Reply:
`groovy.lang.MissingMethodException: No signature of method: java.util.ArrayList.unschedule() is applicable for argument types: (java.lang.String) values: [clearTamper]`

Getting this exception